frontend/public/components/utils/deployment-pod-counts.tsx

```diff
@@ -20,6 +20,21 @@ type DPCState = {
   editable: boolean;
 };
 
+export const checkEditAccess = (resource, resourceKind, impersonate): Promise<SelfSubjectAccessReviewKind> => {
```

Comment:
Better to move this check to a common utility.
But isn't this entire class going away and being replaced by the pod visualization? Please confirm with UX.